src/markdown/tutorial/part-2/09-route-params.md

```diff
@@ -114,43 +114,9 @@ Alright, we have just one more step left here: updating the tests. We can add an
 git add tests/integration/components/rental-test.js
 ```
 
-If we run the tests in the browser, everything should...
+If we run the tests in the browser, everything should just pass!
 
-```run:screenshot width=1024 height=768 retina=true filename=fail.png alt="The test failed"
-visit http://localhost:4200/tests?nocontainer&nolint&deterministic
-wait  #qunit-banner.qunit-fail
-```
-
-...wait a minute, our tests didn't pass!
-
-Well, it's about time that we ran into something that didn't Just Work™ on the first try! This is the *advanced* part of the tutorial after all. 😉
-
-Component tests (like the one we have here) do not set up the router by default, because it's usually not necessary. In this specific case, however, we have a `<LinkTo>` in our component that is relying on the router to generate its URLs.
-
-In this situation, we essentially need to *specifically* opt-in to explicitly use our router in our component test. We can do this by calling `setupRouter()` in our `beforeEach` hook, which will set up the router before each test.
-
-```run:file:patch lang=js cwd=super-rentals filename=tests/integration/components/rental-test.js
-@@ -8,2 +8,6 @@
-
-+  hooks.beforeEach(function () {
-+    this.owner.setupRouter();
-+  });
-+
-   test('it renders information about a rental property', async function (assert) {
-```
-
-> Zoey says...
->
-> As its name implies, the `beforeEach` hook runs *once* before each `test` function is executed. This hook is the ideal place to set up anything that might be needed by all test cases in the file. On the other hand, if you need to do any cleanup after your tests, there is an `afterEach` hook!
```

Comment:
Do we need to move this somewhere else? Do we use beforeEach elsewhere?

Reply:
Good catch! It is indeed taught on this page. I've added it back